pkg/cli/crictl/crictl.go

```diff
@@ -1,11 +1,17 @@
 package crictl
 
 import (
+	"os"
+	"runtime"
+
 	"github.com/kubernetes-sigs/cri-tools/cmd/crictl"
 	"github.com/urfave/cli"
 )
 
 func Run(ctx *cli.Context) error {
+	if runtime.GOOS == "windows" {
```

Comment:
Here you are checking `runtime.GOOS` but in `func runExec` you are adding a variable `calledAsInternal`. I wonder if we could do it the same way in both places

Reply:
@manuelbuil Here the purpose is little bit different from the usage of `calledAsInternal`. For the wrapper of `crictl` or `kubectl`, on the Linux system, the `os.Args` only contains the <additional arguments>. However, on Windows syestem, the `os.Args` contains <executable path> and <additional arguments>`. This patch is to normalize the `os.Args` into the Linux convention so the real module won't be confused by the `os.Args` on Windows.

Reply:
thanks for the explanation